charts/v4.4.0/blob-csi-driver/templates/csi-blob-controller.yaml

```diff
@@ -0,0 +1,224 @@
+kind: Deployment
```

Comment:
could you update the pr description with any changes that went it to the helm charts?